packages/sync/src/links/LinksBuilder.ts

```diff
@@ -31,9 +31,10 @@ export const defaultLinkBuilder: LinkChainBuilder =
     const debounceLink = new DebounceLink();
     let links: ApolloLink[] = [queueMutationsLink, debounceLink, conflictLink(config), httpLink];
 
-    // TODO to be replaced by network interface check when ready
-    NetworkStatus.whenOnline(() => queueMutationsLink.open());
-    NetworkStatus.whenOffline(() => queueMutationsLink.close());
+    const network = new WebNetworkStatus();
```

Comment:
We kinda need singleton for this for network status to be accessible in entire SDK (and also outside module